packages/manager/apps/pci-ai-notebooks/src/hooks/api/ai/authorization/usePostAuthorization.hook.tsx

```diff
@@ -12,17 +13,26 @@ export function usePostAuthorization({
   onError,
   onSuccess,
 }: PostMutateAuthorizationProps) {
+  const { projectId } = useParams();
+  const queryClient = useQueryClient();
   const mutation = useMutation({
-    mutationFn: (projectId: PCIAi) => {
-      return postAuthorization(projectId);
+    mutationFn: () => {
+      return postAuthorization({ projectId });
     },
     onError,
-    onSuccess,
+    onSuccess: (data) => {
+      // invalidate notebook auth
+      queryClient.invalidateQueries({
+        queryKey: [projectId, 'ai/notebook'],
```

Comment:
I think we should invalidate auth key instead

Reply:
Yes you are right. Done